README.md

```diff
@@ -92,60 +92,68 @@ pointed to that branch. Please, make sure you follow the
 require that at least two developers review them first before merging to `dev`
 branch.
 
-### Installing ESP-IDF
+### Installing RIOT-OS
 
-In order to compile the firmware (and to flash it) you need to install the
-[ESP-IDF](https://github.com/espressif/esp-idf/) development framework, which
-brings the compiler and necessary tools in order to correctly build the
-project.
+RIOT is a real-time multi-threading operating system that supports a range of devices that are typically found in the Internet of Things (IoT): 8-bit, 16-bit and 32-bit microcontrollers.
 
-Please note that we use a fork of the ESP-IDF, that enables us to use SLIP as
-a network interface (SLIP ESP-NETIF) until we get upstream support, it can
-be used as follows:
+### Pre-requisites
 
+1. Install Git.
+2. Install the build-essential packet (make, gcc etc.). This varies based on the operating system in use. para `mac` le recomendamos usar `xcode-select install`.
+3. Install OpenOCD
```

Comment:
- don't mix languages
- show a more detailed process and don't use things like **etc** .
- put a link to the official RIOT-OS documentation for this process.